### PR TITLE
Changes for admin portal lists

### PR DESCRIPTION
**Pull request by BradHN1**
Issue 549 - List length default 25 on all pages, with options [10, 25, 100]
Issue 548 - add ID column back on most lists
Issue 542 - display date in sortable format (YYYY-MM-DD)

### CODE REVIEW

**Review by frimpongopoku — APPROVED** [review]
LGTM

File: app/utils/common.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import qs from "qs";
 export const getHumanFriendlyDate = (dateString, includeTime = false) => {
   if (!dateString) return null;
   return moment(dateString).format(
-    `MMMM Do, YYYY ${includeTime ? "hh:mm a" : ""}`
+    // make it a bit less human friendly, so it sorts properly
+    `YYYY-MM-DD ${includeTime ? "hh:mm a" : ""}`
+    //`MMMM Do, YYYY ${includeTime ? "hh:mm a" : ""}`
```
Comment (Davidjwkinghorn) [inline app/utils/common.js:12]: We should revisit this later. User facing dates should be as human friendly as possible. Ideally we should sort by date objects if that's possible.

**Review by Davidjwkinghorn — APPROVED** [review]
LGTM

**Comment by Davidjwkinghorn** [comment]
Why were the relevant issues closed before this has been merged?

**Comment by BradHN1** [comment]
Our process is the developer should close the issue when he/she thinks it is coded and working, ready to deploy.  It can be reopened when we decide it isn’t ready.  We could change that process but this works fairly well.

**Comment by Davidjwkinghorn** [comment]
That makes sense. 

I'm asking because I'm used to developers moving tickets between columns on the project board to indicate the status (Moving a ticket to QA would indicate it's ready for review and testing). In that case issues would only be closed when the feature, bug fix, epic, etc is deployed to prod.

**Comment by Davidjwkinghorn** [comment]
I'm thinking of rolling this out with [issue 547](https://github.com/massenergize/frontend-admin/pull/547), unless it's going to be a little while.

File: app/utils/common.js (hunk 1 above)
Comment (BradHN1) [reply app/utils/common.js:12]: We could improve maybe but this just admins and the list view is very space constrained.

**Pull request merged**